integrated_channels/canvas/docs/decisions/0001-integration-classic-vs-lti.rst

```diff
@@ -0,0 +1,37 @@
+Canvas Integration via Classic Integration model vs LTI
+=======================================================
+
+Status
+------
+
+Draft
```

Comment:
I think we typically release these as "Accepted". I'd also move this to https://github.com/edx/edx-enterprise/tree/master/docs/decisions with the other docs.